<launch>
	<arg name="show_image" default="1" />
    <rosparam file="$(find ec_at_va)/cfg/mask_param.yaml" />
    <node name="get_point_message" 
        pkg="ec_at_va" 
        type="get_point_from_message.py"
        output="screen"
        args="$(arg show_image)">
	    <remap from="image_raw" to="/camera/rgb/image_raw"/>
	    <remap from="goal_base" to="/camera/img_base"/>
	</node>
</launch>
